### PR TITLE
NO_ISSUE: add new pipeline to test manual with label sno and compact

### PR DESCRIPTION
**Pull request by alknopfler**
Signed-off-by: Alberto Morgante Medina <alknopfler@users.noreply.github.com>

# Description

add new pipline to test manual with label sno and compact, removing the approve

The idea about testing ci PR will be:
- New labels will be created: "test-sno" "test-compact" and will be removed the current "test-ci" 
- PR owner is responsible to pass at least 1 test using "test-sno" and/or "test-compact" for each PR depending of the feature to be merged.
- We MUST showing the result in the PR description as a warranty of success for the rest of PR reviewers. If there's a test we cannot merge the PR to ensure every PR is tested.
- No automated test will be launched with the approval
- If you need add more commits in the PR but you don't want to pass new tests, you need to remove the label before if you don't want to pass the test again. Makes sense that if you add something relevant for the PR, and the label is set, then it will pass again the tests automatically (this is the behaviour if you don't remove the label before committing new things). 

Fixes # NO_ISSUE: PR to get agreement with testing ci strategy

## Type of change

Please select the appropriate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Review by flaper87 — CHANGES_REQUESTED** [review]

File: .github/workflows/testing-pr-sno.yml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+          fi
+
+          echo ">>>> Launching Pipeline edgecluster"
+          echo "Release: ${RELEASE}"
+          echo "Image Branch: ${{ github.event.pull_request.head.ref }}"
+          echo "Git hash: ${{ github.sha }}"
+          echo ">>>>"
+
+          BRANCH=${{ github.event.pull_request.head.ref }} make bootstrap
+          RELEASE=${RELEASE} make build-edgecluster-sno-2nics
```
[the remark this answers is not in the document]
Comment (alknopfler) [reply .github/workflows/testing-pr-sno.yml:125]: yes we can, but I'm thinking if also is interesting have the 4 possibilities:
- test-sno-1nic
- test-sno-2nics
- test-compact-1nic
- test-compact-2nics
or maybe too complex? WDYT

if not we could get a mix:
- test-sno (with 1nic)
- test-compact (with 2nics)

[the remark this answers is not in the document]
Comment (flaper87) [reply .github/workflows/testing-pr-sno.yml:125]: right, compact is already testing 2nics. I was referring only to the SNO test. Let's make it 1nic

[the remark this answers is not in the document]
Comment (alknopfler) [reply .github/workflows/testing-pr-sno.yml:125]: ok, then
- test-sno (with 1nic)
- test-compact (with 2nics)

**Review by iranzo — APPROVED** [review]
lgtm

**Review by flaper87 — APPROVED** [review]

**Pull request merged**